src/platform/qt/AssetTile.cpp

```diff
@@ -32,7 +32,7 @@ AssetTile::AssetTile(QWidget* parent)
 	m_ui.color->setDimensions(QSize(1, 1));
 	m_ui.color->setSize(50);
 
-	connect(m_ui.preview, SIGNAL(indexPressed(int)), this, SLOT(selectColor(int)));
+	connect(m_ui.preview, &QGBA::Swatch::indexPressed, this, &AssetTile::selectColor);
```

Comment:
Is there a conflict that is preventing this from just being `Swatch::` instead of `QGBA::Swatch::`?